.travis.yml

```diff
@@ -5,3 +5,7 @@ services:
 
 script:
   - make build test
+
+after_success:
+  - make coverage
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
I agree, however, bash version seems to be more language agnostic. All in all, this is just for evaluation purposes.